src/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
+  try {
+    const serializedState = localStorage.getItem('state');
```

Comment:
An improvement item: the string 'state' probably is better off assigning it to an exportable constant variable and be access through the variable i.e. `localStorage.getItem(USER_STATE_KEY)`